aries-site/src/pages/templates/empty-state.mdx

```diff
@@ -0,0 +1,130 @@
+
+1.  **Clear messaging:** The empty state should have simple and clear messaging that tells the user what information is missing, why it is missing, and what they can do next to access the information. Follow the voice and tone guidance within the design system to remain clear and empathetic to the user.
+1.  **Visual cues:** The empty state should use iconography to help convey the message and create a more engaging user experience. The icon should be relevant to the context of the interface and help the user understand why it's empty.
+1.  **Guidance:** The empty state should provide guidance and a direct pathway out of the empty state when applicable. Avoid leading the user to a dead end.
+1.  **Calls-to-action:** The empty state should include calls-to-action that encourage the user to take action, such as creating their first resource or adding their first item.
+1.  **Help or documentation links:** The empty state may include links to help or documentation resources that can assist the user in populating the view with data or finding relevant information.
+
+## Anatomy
+
+<Example plain>
+  <EmptyStateAnatomy />
+</Example>
+
+Although there are many diverse use cases for empty states within our solutions, we use a single component structure to ensure they are consistently applied across the complete user experience. Below describes the structure of an empty state component:
+
+| Label | Region          | Purpose                                                             |                  Required                   | Notes                                                                      |
+| ----- | --------------- | ------------------------------------------------------------------- | :-----------------------------------------: | -------------------------------------------------------------------------- |
+| **1** | **Icon**        | Imagery to help the user process the message.                       |                                             | Use size="xlarge" on icon.                                                 |
+| **2** | **Heading**     | Communicates the system status.                                     | <Checkmark a11yTitle="true" size="small" /> | Use the semantic heading level that fits the placement of the empty state. |
+| **3** | **Description** | Guides the user out of the empty state                              | <Checkmark a11yTitle="true" size="small" /> | Use default text size of "medium".                                         |
+| **4** | **Action**      | A direct pathway out of the empty state.                            |                                             |                                                                            |
+| **5** | **Anchor**      | An additional learning cue to help the user out of the empty state. |                                             |                                                                            |
```

Comment:
I know we've been leaning towards using the term 'Link' instead of 'Anchor' ... should that be changed here as well?

Reply:
I might've missed that -- happy to adjust if that's the desired direction. Can you fill me in on when we shifted towarads that? The reason I used "Anchor" is because that aligns with the component name in Grommet and Figma.

Reply:
As well as semantic HTML.

Reply:
@SOjaHPE any thoughts you can weigh in on here? I know we've discussed in the past how the term 'Link' might better fit a user's mental modal as opposed to 'Anchor'

Reply:
Some additional context from the grommet side... Eric and I have chatted about renaming the grommet component from 'Anchor' to 'Link'. This is the direction we would like to eventually head, but for backwards compatibility we couldn't make this change to the grommet component without doing a major version release. It is tracked in [this issue](https://github.com/grommet/grommet/issues/3297) as well as in our [grommet v3 wiki](https://github.com/grommet/grommet/wiki/Grommet-v3).
Totally fine if we want to refer to it as Link here, just wanted to give some background from the grommet side.

Reply:
Changed to Link.